libexec/rtld-elf/riscv/rtld_start.S

```diff
@@ -104,6 +104,49 @@ ENTRY(.rtld_start)
 #endif /* defined(__CHERI_PURE_CAPABILITY__) */
 END(.rtld_start)
 
+#define	INT_SIZE	8
+#define	INT(x)		x
+#define	LOAD_INT	PTR(ld)
+#define	STORE_INT	PTR(sd)
```

Comment:
Should we share this with e.g. setjmp?

Reply:
We could but I'd start to worry about conflicts for `INT` and `CAP` if putting them in asm.h, though I guess if guarded with `_LOCORE` it should be fine.